.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+# Javascript Node CircleCI 2.0 configuration file
```

Comment:
@mjfrancoeur You can move this `circleci/config.yml` and the `.eslintrc.json` files into separate PRs.

```
git checkout master
git checkout feature/add-linting
git checkout feature/USDA-api-call -- .eslintrc.json
```

Then push and create a PR for `feature/add-linting`. Then do something similar for circleci.